server/routerlicious/packages/memory-orderer/src/localOrderer.ts

```diff
@@ -343,15 +343,19 @@ export class LocalOrderer implements IOrderer {
             this.tenantId,
             this.documentId,
             this.gitManager,
+            null,
```

Comment:
>null [](http://example.com/codeflow?start=12&length=4)

(nit) include the parameter name in a comment (/* deltaService */) for a better readability?

Reply:
Done